cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -280,6 +282,8 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
           |For more details on the differences between graphs, databases and the DBMS, refer to <<neo4j-databases-graphs>>.""".stripMargin)
       p("image::grant-privileges-dbms.png[title=\"Syntax of GRANT and DENY DBMS Privileges\"]")
       // image source: https://docs.google.com/drawings/d/1UAyLvL7UdwYM1I9RVrwxnvWhB4zSu9cv4rCzzqC3-iU/edit?usp=sharing
+      p("image::privilege-hierarchy-dbms.png[title=\"Dbms privileges hierarchy\"]")
```

Comment:
As discussed, the text of this image comes out very small when it's scaled to fit into the layout.

Reply:
Since you can click it up to fullsize it, this might not be that big of an issue but it might be nice to get some feedback from the documentation team on this.

It is not that easy to make another layout that both looks good and still shows things reasonably collected. So unless it is a problem that it gets so small by default I prefer to leave it as it is now.